doc/api/net.md

```diff
@@ -185,12 +185,11 @@ by the OS through sysctl settings such as `tcp_max_syn_backlog` and `somaxconn`
 on Linux. The default value of this parameter is 511 (not 512).
 
 
-Note:
+**Note:** All [`net.Socket`][] are set to `SO_REUSEADDR` (See [socket(7)][] for
+details).
 
-* All [`net.Socket`][] are set to `SO_REUSEADDR` (See [socket(7)][] for
```

Comment:
Nit: I understand this goes against the style defined above but isn't it better to keep the bulleted list here instead of repeating **Note:** in two consecutive lines?